driver-core/src/main/java/com/datastax/driver/core/policies/SpeculativeExecutionPolicy.java

```diff
@@ -64,8 +64,8 @@
          * Returns the time before the next speculative query.
          *
          * @param lastQueried the host that was just queried.
-         * @return the time (in milliseconds) before a speculative query is sent to the next host. If zero or negative,
-         * no speculative query will be sent.
+         * @return the time (in milliseconds) before a speculative query is sent to the next host. If negative,
+         * no speculative query will be sent. If zero it will immediately send the retry.
```

Comment:
Nit: let's use the term "speculative (execution|query|request)" and avoid the term "retry" so that users do not confuse this with retry policies :)